CHANGES.md

```diff
@@ -1,6 +1,473 @@
 Release Changes List
 ====================
 
+version 1.17.0
+==============
+
+Twentieth public release of Chapel, April 5, 2018
+
+Highlights (see subsequent sections for further details)
+--------------------------------------------------------
+* tool / configuration highlights:
+  - added support for bash autocompletion for `chpl` compiler options
+  - `chpl` now offers suggestions for unrecognized flags
+  - changed the generated executable's name from the main module to its filename
+  - improved support for `chpl`s LLVM back-end
+  - added support for multiple mason repositories, including local registries
+* language highlights:
+  - dramatically improved support for user-defined and implicit initializers
+  - added support for uninterpreted triple-quoted strings
+  - notable improvements to `Owned(c)` and `Shared(c)`
+* standard library/package highlights:
+  - `LinearAlgebra` matrices now use 1-based indices by default
+  - new Kronecker, Cholesky, eigenvector/value routines in `LinearAlgebra`
+  - added the Blowfish cipher to the `Crypto` module
+  - added support for joinPath(), isAbsPath(), commonPath to the `Path` module
+  - added Bessel fucntions to `Math` module
+  - added new channel.advancePastByte(), mark(), commit(), and revert() methods
+  - increased the use of error-handling in standard, package, internal modules
+* performance / benchmark / memory highlights::
+  - significant performance improvements when executing using Cray networks
+  - added a new faster version of the reverse-complement benchmark
+  - closed a number of minor memory leaks
+* portability highlights:
+  - ported Chapel to the Cray XC50 system with ARM processors
+  - improved portability to FreeBSD and PowerPC
```

Comment:
I'm not sure where PowerPC came from... I helped with FreeBSD and NetBSD. I don't see 'Power' in the commit history since I mentioned it in 2014 :)

Reply:
Cool, looks like I just needed to search for `powerpc`.